examples/30_stereo_depth_from_host.py

```diff
@@ -170,6 +170,7 @@ def convert_to_cv2_frame(name, image):
                 img.setData(data)
                 img.setTimestamp(tstamp)
                 img.setInstanceNum(inStreamsCameraID[i])
+                img.setType(dai.RawImgFrame.Type.RAW8)
```

Comment:
Change to:
```
dai.ImgFrame.Type.RAW8
```

Reply:
Changed, everywhere in the examples where it was used. So it's consistent.